salt/loader.py

```diff
@@ -125,6 +125,19 @@ def minion_mods(
                      pack={'__context__': context},
                      whitelist=whitelist,
                      loaded_base_name=loaded_base_name)
+
+    # Load any provider overrides from the configuration file providers option
+    #  Note: Providers can be pkg, service, user or group - not to be confused
+    #        with cloud providers.
+    if opts.get('providers', False):
+        providers = opts.get('providers', False)
```

Comment:
The default here should be an empty list-- otherwise for mod in False will raise an exception